app/controller/auth.js

```diff
@@ -36,71 +36,80 @@ app.use(passport.session());
 // Express routes for authentication
 // Redirect to GitHub to login
 app.get('/auth/github',
-  passport.authenticate('github', { scope: 'repo read:org' }),
-    function(req, res) {
-      // Just a redirect, no actions here.
-    });
+  passport.authenticate('github', {
+    scope: 'repo read:org',
+  }), (req, res, next) => {
+    return next(); // Never gets called
```

Comment:
Because it always gets redirected to GitHub for authentication, which then gets redirected to the callback URL.